_layouts/page.html

```diff
@@ -48,6 +48,7 @@ <h2>{{ item.title }}</h2>
   <title>{% if site.xdmod_module_name %}{{site.xdmod_module_name}} {{doc_path}}{% else %}Open XDMoD{% if page.title != 'Open XDMoD' %} - {{ page.title }}{% endif %}{% endif %}</title>
   <link rel="icon" type="image/x-icon" href="{{ site.baseurl }}{{ site.theme_subdir }}/assets/images/favicon.ico" />
   <link rel="stylesheet" type="text/css" href="{{ site.baseurl }}{{ site.theme_subdir }}/assets/css/{{style_sheet}}.css" />
+  {% unless page.version==site.latest_version %} <link rel="canonical" href="https://open.xdmod.org/{{site.latest_version}}/{{page.name | remove: '.md' }}.html" /> {% endunless %}
```

Comment:
Please can you change the hardcoded "https://open.xdmod.org" to the variable that has the domain name. These templates are used for all the various sites supremm.xdmod.org, appkernels.xdmod.org etc...

Reply:
I just made this change, it should be the second commit in this branch.